chunky/src/java/se/llbit/chunky/renderer/scene/PreviewRayTracer.java

```diff
@@ -110,18 +110,23 @@ public static boolean nextIntersection(Scene scene, Ray ray) {
   }
 
   private static boolean waterIntersection(Scene scene, Ray ray) {
+    double t = (scene.getEffectiveWaterPlaneHeight() - ray.o.y - scene.origin.y) / ray.d.y;
+    if (scene.waterPlaneClip) {
```

Comment:
Could this use a getter? 🙈

Reply:
This calculates `t` if `ray.d.y == 0`, which is not needed. I guess that never really happens a lot though.